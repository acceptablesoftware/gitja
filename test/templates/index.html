{% extends "title.html.include" %}
{%- block title -%}{{ host }}{%- endblock -%}
{% block body %}
{{ repositories }}
{% endblock %}
