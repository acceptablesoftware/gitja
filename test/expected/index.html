https://github.com/m-col/gitserve
gitserve
